example/android/shared/src/commonMain/kotlin/com/myrnproject/shared/E2ETest.kt

```diff
@@ -83,7 +84,7 @@ class E2ETest {
 
 
     @ReactNativeMethod
-    fun testKotlinDateTime(instant: Instant, localDateTime: LocalDateTime): Duration {
+    fun testKotlinDateTime(instant: Instant, date: @JSType("date") Instant,localDateTime: LocalDateTime, test: DateTimeTest): Duration {
```

Comment:
I would prefer Date to be the default but we can also leave it this way

Reply:
it is now possible to set the default via ksp options